libsolidity/formal/CHC.cpp

```diff
@@ -1807,6 +1807,14 @@ tuple<CheckResult, smtutil::Expression, CHCSolverInterface::CexGraph> CHC::query
 	CheckResult result;
 	smtutil::Expression invariant(true);
 	CHCSolverInterface::CexGraph cex;
+	if (m_settings.printSmt)
+	{
+		string smtLibCode = m_interface->getSMTLIBCode(_query);
+		m_errorReporter.info(
+			2339_error,
+			"CHC: SMTLIB code used for formal verification:\n" + smtLibCode
```

Comment:
```suggestion
			"; CHC: Requested query:\n" + smtLibCode
```
The `;` here is to add this as a comment if people dump the output directly to an SMT file